front/src/modules/pathfinding/components/Pathfinding/TypeAndPathV2.tsx

```diff
@@ -223,6 +223,12 @@ const TypeAndPathV2 = () => {
     setInitialCursorPositionRem(0);
   }, []);
 
+  const isSortedSearchResultsDisplayed = () => {
```

Comment:
I would rather use a useMemo here so `isSortedSearchResultsDisplayed` is a const and not a function, can you change that ? :)

Reply:
Done :)